pkg/storage/mongo/getter.go

```diff
@@ -44,24 +29,11 @@ func (s *ModuleStore) Info(ctx context.Context, module, vsn string) ([]byte, err
 // GoMod implements storage.Getter
 func (s *ModuleStore) GoMod(ctx context.Context, module, vsn string) ([]byte, error) {
 	const op errors.Op = "mongo.GoMod"
-	ctx, span := observ.StartSpan(ctx, op.String())
```

Comment:
Please leave it as it is to collect traces?

Reply:
No problem. Thanks for taking the time to review these changes!